test/frameworks/grape_test.rb

```diff
@@ -22,19 +22,19 @@
       r.headers.key?('X-Trace').must_equal true
 
       traces = get_all_traces
```

Comment:
I like this idea. Node testing doesn't do it this way but there are a lot of benefits to getting all traces at one time.

Reply:
;) it works well, I didn't write it though.